app/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ plugins {
     alias(libs.plugins.anvil)
     alias(libs.plugins.ksp)
     alias(libs.plugins.kapt)
-    alias(libs.plugins.firebaseAppDistribution)
+    id(libs.plugins.firebaseAppDistribution.get().pluginId)
```

Comment:
Why this change?

Reply:
Otherwise there is an issue with the firebase BOM, it keeps throwing an error about the dependency already existing in the classpath with another version or something similar.